package.json

```diff
@@ -5,8 +5,8 @@
     "url": "http://gajus.com"
   },
   "dependencies": {
-    "hammerjs": "^2.0.4",
-    "lodash": "^4.6.1",
+    "angular": "^1.5.4",
+    "angular-touch": "^1.6.1",
```

Comment:
Should probably not mix-n-match these, even if the `^` means they'll both end up being 1.6.x